core/block_svg.js

```diff
@@ -1632,7 +1632,77 @@ class BlockSvg extends Block {
    * @override
    */
   getParent() {
-    return /** @type {!BlockSvg} */ (super.getParent());
+    return /** @type {?BlockSvg} */ (super.getParent());
+  }
+
+  /**
+   * @override
```

Comment:
There is no style guide rule about it, but I think we generally put the @override after @return.